### ISSUE TITLE
iOS Support?

### ISSUE THREAD

**Issue by peardox** [Embarcadero/Lightweight-Python-Wrappers#6]
Just found this...

https://github.com/beeware/Python-Apple-support

Which sorta implies Delphi Python should also work?

**Comment by lmbelo** [comment]
Hi @peardox,

Python might work on iOS, but I think we can deploy our apps to the App Store due to run-time code changes.

Have a look in the MethodCallback.pas unit at P4D repo.

**Comment by peardox** [comment]
For reference I got in this track owing to a youtube (don't worry, it's short and relevant)

https://www.youtube.com/watch?v=VX8gNHCDA3Q

The app he shows is, he says, in Python and it's on the Apple Stores - I've got it on my iPad and it does work.

My note above was the result of a little research and looked the most pertinant from my brief look into the topic

**Comment by lmbelo** [comment]
His app might not change run-time code.

**Comment by peardox** [comment]
Yeah, I doubt it. He'll be using some pre-packager. The packager may however change runtime.

If making an iOS ARM64 "Hello World" isn't too much work though it'd be interesting to see what happens if it's put through release submission (I've got an Apple Dev account valid til Dec if needed)

**Comment by lmbelo** [comment]
I also have this on the roadmap. I will try this as I get an opportunity and a new phone.
